foreshadow/wrapper.py

```diff
@@ -101,8 +102,10 @@ def __init__(self, *args, **kwargs):
             try:
                 super(DFTransformer, self).__init__(*args, **kwargs)
             except TypeError as e:
-                raise type(e)(str(e) + ". Init for transformer: '{}' "
-                                          "called".format(transformer))
+                raise type(e)(
+                    str(e) + ". Init for transformer: '{}' "
+                    "called".format(transformer)
+                )
 
             # TODO: remove this when _Empty is removed
             self.__empty_fit = False
```

Comment:
Now that we turned off Preprocessor, can this be removed?

Reply:
Yes